pkg/controller/environmentApi.go

```diff
@@ -238,3 +239,40 @@ func (a *API) EnvironmentApiDelete(w http.ResponseWriter, r *http.Request) {
 
 	a.respondWithSuccess(w, []byte(""))
 }
+
+// EnvironmentApiPodList: Get list of pods currently available in environment
+func (a *API) EnvironmentApiPodList(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	name := vars["environment"]
+
+	// label selector
+	selector := map[string]string{
+		fv1.ENVIRONMENT_NAME: name,
+	}
+
+	ens := a.extractQueryParamFromRequest(r, "environmentnamespace")
```

Comment:
Please define constants for these, instead of hardcoding.
Same constant can be used in the client and API sides.
`environmentnamespace`, `executortype`
Same for other query parameters.

Reply:
Fixed